### PR TITLE
Create new use-cases

### PR DESCRIPTION
**Pull request by mdoucet**
Provide the three main use-case for RefRed. They describe the main activities that users and staff will do with the application.

### CODE REVIEW

**Comment by mdoucet** [comment]
@wdzhou Ready for you.

**Comment by wdzhou** [comment]
> @wdzhou Ready for you.

Thanks.  Will start the review soon.

File: docs/compute-scaling-factors.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Compute Scaling factors
+-----------------------
+
+1- In the `Tools` menu, select `SF Calculator`.
+A new window will pop-up.
+
+2- The new window only has one activated field to enter a range of run
+numbers. Enter the following run range: `184975-184989` and click enter.
+
```
Comment (wdzhou) [inline docs/compute-scaling-factors.rst:9]: It will be nice to have a screenshot here and point out that the loading could take more than a few seconds.

**Review by wdzhou — COMMENTED** [review]
Overall, the use cases look good to me.

**Review by wdzhou — APPROVED** [review]
@mdoucet I left some comments on your use cases.  Overall, I think they are in good shape and thus approved.

File: docs/compute-scaling-factors.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+3- The table will be populated with all the runs in the selected range.
+For each run, the peak will be selected. You can browse through the runs
```
[the remark this answers is not in the document]
Comment (mdoucet) [reply docs/compute-scaling-factors.rst:11]: I modified to text to say it's done automatically.

File: docs/compute-scaling-factors.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+to verify that the selection was good.
+
+4- Using the browse button, you can select the location of the output file
+that will be generated.
+
+5- Users will typicalling not use the `manual` radio button, or the `Sort`
```
[the remark this answers is not in the document]
Comment (mdoucet) [reply docs/compute-scaling-factors.rst:17]: Clarified the text.

File: docs/reduce-data.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
+Reducing data using RefRed
+--------------------------
+
+This use-case describes how the user interacts with RefRed once their reduction
+options have been chosen.
+
+
+1- Load a template according to the corresponding user case.
```
[the remark this answers is not in the document]
Comment (mdoucet) [reply docs/reduce-data.rst:8]: Done

File: docs/reduce-data.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+2- Click the "reduce" button at the bottom of the interface.
+   Once the reduction starts, the user will be brought to the "data stitching"
+   tab (see image). As the reduction proceeds, each data set will be added
+   to the plot (item A).
+
+.. image:: reduction-results.png
```
[the remark this answers is not in the document]
Comment (mdoucet) [reply docs/reduce-data.rst:15]: Added a link to the use-case that contains an image showing the reduce button.

**Review by wdzhou — APPROVED** [review]
It looks good to me.

**Pull request merged**